apps/address-service/domains/address/schema/Address.js

```diff
@@ -34,6 +36,28 @@ const Address = new GQLListSchema('Address', {
             isRequired: true,
         },
 
+        overrides: {
+            schemaDoc: `The list of overrides for address ${OVERRIDING_ROOT} field`,
+            adminDoc: `This is a JSON object that must look like {"field":"value"}. Will use to override ${OVERRIDING_ROOT}`,
+            type: Json,
+            isRequired: false,
+            access: {
+                create: access.canManageOverrides,
+                update: access.canManageOverrides,
+            },
+            hooks: {
+                validateInput: async (data) => {
+                    const { resolvedData, addFieldValidationError, existingItem, fieldPath } = data
+                    Object.entries(get(resolvedData, fieldPath, {}) || {}).forEach(([path, value]) => {
+                        if (!has({ ...existingItem, ...resolvedData }, `${OVERRIDING_ROOT}.${path}`)) {
```

Comment:
I cannot add field to meta here?

Is it possible, that dadata returns `{a: 1, b: 2}` for 1 object, and `{a: 2}` for second? Or it will always be `b: null` in here?

Reply:
Yes, no new fields. Only overriding existing ones. If new fields will make sense I'll add the functionality.

Reply:
And this is already normalized data, so all fields will be there.